### ISSUE TITLE
Variables in config

### ISSUE THREAD

**Issue by smdremedy** [byoutline/MockServer#3]
Paths, queries and headers should allow using simple variables e.g.
in main config:
"auth" = "auth-token-12345"

```
in request
{
  "headers" : {
     "Auth-Token" : "${auth}"
  }
}
```

**Comment by Naitbit** [comment]
Can we use similar mechanism to `"response file"` instead? We would not have a variable, but we would have file that could be reused in multiple requests.
Additionally we could override/attach content of the file using old `"headers"` syntax like this:

``` json
"headers file": "commonHeaders.json"
"headers" : {
     "Auth-Token" : "auth-token-12345"
  }
```

This syntax could be also added for `"queries"` as `"queries file"`.

**Comment by Naitbit** [comment]
For now I've added support for `"headers file"` and `"queries file"`. ( 568768701a3121abf2afb55041f5f0b68101a2ac ). If it is not sufficient please reopen this issue.

**Comment by Naitbit** [comment]
Available for testing in 2.0.0-SNAPSHOT
